bazel/api-gen/rendering/templates/cli-reference.tsx

```diff
@@ -73,3 +53,12 @@ export function CliCommandReference(entry: CliCommandRenderable) {
     </div>
   );
 }
+
+
+function commandName(entry: CliCommandRenderable, command: string) {
+ if(entry.parentCommand?.name) {
```

Comment:
Looks like this is missing a space?
```suggestion
 if (entry.parentCommand?.name) {
```

Reply:
✅